extlinks/links/admin.py

```diff
@@ -23,10 +24,16 @@ class LinkEventURLPatternAdminInline(GenericTabularInline):
         "on_user_list",
     ]
 
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+
+        return qs.select_related("username")
+
 
 class URLPatternAdmin(admin.ModelAdmin):
     list_display = ("url",)
     exclude = ["collections"]
+    autocomplete_fields = ["collection"]
```

Comment:
Is this allowing you to filter link events by collection locally? When I update the collection nothing seems to happen.

Reply:
It uses a `Select2` autocomplete field that loads the collection data asynchronously. I am seeing it works on my local environment. Do you see any console errors?